### PR TITLE
Examples fixes

### PR DESCRIPTION
**Pull request by vvhsk**
Couldn't start demos from project without build because no files in /dist, but index.html refers to it.
Also couldn't see any publishers if http server is started not in the root project directory.

Thinking that it should be mentioned in demos readme.

### CODE REVIEW

File: example/browser/basic/README.md [hunk 1]
```diff
@@ -2,7 +2,13 @@
 
 ### Usage
 
-Host `index.html` with any server and open it in your browser.
+Install dependencies and build app
```
Comment (lahmatiy) [inline example/browser/basic/README.md]: Suggestion:
```
Make sure dependencies are installed and rempl is built
```

File: example/browser/basic/README.md [hunk 2]
```diff
@@ -2,7 +2,13 @@
...
+
+```
+npm install && npm run-script build
+```
+
+Host `index.html` from the project root directory with any server and open it in your browser.
```
Comment (lahmatiy) [inline example/browser/basic/README.md]: Suggestion:
```
Start any http-server in project's root and open `example/basic/index.html` in your browser.
```

File: example/browser/basic/README.md [hunk 3]
```diff
@@ -2,7 +2,13 @@
 
 ### Usage
 
-Host `index.html` with any server and open it in your browser.
+Install dependencies and build app
+
+```
+npm install && npm run-script build
```
Comment (lahmatiy) [inline example/browser/basic/README.md]: `run-script` -> `run`

File: example/browser/basic/README.md [hunk 4]
```diff
@@ -2,7 +2,13 @@
...
+```
+
+Host `index.html` from the project root directory with any server and open it in your browser.
 
 Open rempl-server page in you browser ([http://localhost:8177/](http://localhost:8177/) by default) and choose `web-input`-publisher.
```
Comment (lahmatiy) [inline example/browser/basic/README.md]:
```
Start [`rempl` server](https://github.com/rempl/rempl-cli), open its host in your browser ...
```

**Comment by lahmatiy** [comment]
@vvhsk Are you planning to fix the PR?

**Comment by vvhsk** [comment]
@lahmatiy thanks for suggestions and sorry for delay. 

Is general idea of basic example is publishing date and time and not in listening for 'provide-test' element changes?  Noticed that there is no explanation when to use pick a publisher button. Is it should be in readme?

**Comment by lahmatiy** [comment]
@vvhsk Oops, that's the experiments leaked to repo. I'll fix it later. Thank you for notice.

**Comment by lahmatiy** [comment]
Thanks!

**Pull request merged**